src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,11 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Interface for printing Name, Phone, Email, Address
```

Comment:
this comment is quite off the point, no?
any class that implements this interface can provide the printable string form of the data contained in it.
header comment need to be precise for interfaces and abstract classes, as these form a contract that you are providing to client code developers